viewer/views.py

```diff
@@ -364,6 +381,42 @@ def get_serializer_class(self):
             return SnapshotReadSerializer
         return SnapshotWriteSerializer
     filter_class = filters.SnapshotFilter
-    # filter_permissions = "target_id__project_id"
-    # filter_fields = '__all__'
 ### End of Session Project
+
+
+class TargetCompoundSetsView(views.APIView):
+    def get(self, request, targetID):
+        target = Target.objects.get(id=targetID)
+        compound_sets = CompoundSet.objects.filter(target=target)
+        a_compound_set = compound_sets[0]
+        compound_mols = ComputedCompound.objects.filter(compound_set=a_compound_set)
+        numerical_scores = NumericalScoreValues.objects.filter(compound=compound_mols[0])
```

Comment:
Again, here you're returning the scores just for the first compound set (see other comment above)